### PR TITLE
MVP of including component usage.md files in the documentation

### PR DESCRIPTION
**Pull request by ascott1**
This is an MVP of getting the `usage.md` files moved into the documentation.

What this does:

- Adds a gulp task that takes the component usage.md and converts it to html so that it can be imported into a Jekyll template
- Does the importing and sets up navigation for those items
- Adds the cf `js` file so that the expandables demo works
- Adds dummy classes for the `cf-grid` demo
- Shortens the text on the landing page (scope creep, but ever since @cfarm mentioned it, it's really been bugging me)

## Demo

https://ascott1.github.io/capital-framework/
https://ascott1.github.io/capital-framework/components/

## Review

Any two of the following: @contolini @Scotchester @KimberlyMunoz @jimmynotjim @cfarm

### CODE REVIEW

**Comment by ascott1** [comment]
Besides fleshing out the usage guides, these will need:

- moar betterer design of the component pages
- syntax highlighting

**Comment by Scotchester** [comment]
@ascott1 Do you want to add those to-do items, or go ahead and merge this?

File: components/cf-core/index.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+layout: default
+title:  "cf-buttons"
```
Comment (contolini) [inline components/cf-core/index.md]: Should be `cf-core`.

**Comment by ascott1** [comment]
@Scotchester I would be in favor of merging this and addressing those in future PR's.

**Comment by Scotchester** [comment]
:+1:

**Comment by Scotchester** [comment]
selfie-0

**Pull request merged**